### ISSUE TITLE
Yoshi's Island has line issues with 3D objects

### ISSUE THREAD

**Issue by 03bgood** [bubble2k16/snes9x_3ds#69]
I'm using the most current version of SNES9X on my New 3DS (which is 1.20) and YI runs fine except for one issue; during stages with 3D objects (mainly fort and castle stages, like the spinning log platforms) there seems to be a rendering issue where some lines show up in the background. They're usually red and yellow ones. This issue wasn't present in 1.11 but somehow is in 1.20. Is there any way to fix this? This is the only ROM giving me GFX issues. SMRPG, KDL3, KSS, and SF don't have any GFX issues, as long as the "enable mid frame palette changes" option is selected yet YI still does, regardless of the situation.

**Comment by bubble2k16** [comment]
Strange... I was pretty sure v1.11 had that problem too. I see vertical lines or diagonal lines appearing, but I have no idea why...

**Comment by 03bgood** [comment]
Never noticed it in 1.11, though.

**Comment by bubble2k16** [comment]
Yup I just tested it in the Windows version of Snes9x 1.43. Seems like all these lines have been there all along. It's probably an underlying problem with Snes9x 1.43... Nothing much I can do to fix this... unless I know what the real problem is. :)

**Comment by 03bgood** [comment]
I know Star Fox also has 3D objects and uses the same chip as YI, yet I don't see any issues with SF, yet I've never really played it that much to even notice. I've played YI several times to know how well it emulates.

If I recall, in 1.11, entering through doors makes a lagging stuttering sound, which the original game always makes a shuttering sound, but it slightly lags on this emulator, yet the problematic rendering lines near 3D objects are nowhere to be found.
